contracts/smart-account/modules/SessionKeyManagerModule.sol

```diff
@@ -34,6 +46,7 @@ contract SessionKeyManager is BaseAuthorizationModule {
      */
     function setMerkleRoot(bytes32 _merkleRoot) external {
         userSessions[msg.sender].merkleRoot = _merkleRoot;
+        // TODO:should we add an event here? which emits the new merkle root
```

Comment:
Not sure how helpful it will be. give me an example where this merkle root alone will be helpful

Reply:
This is the link to the discussion on this matter, I explain there the potential use case for such an event

https://biconomyworkspace.slack.com/archives/C05E6G3NTAN/p1691566571842799

please let me know wdyt